cmd/dependencyverifier/dependencyverifier.go

```diff
@@ -26,7 +26,7 @@ import (
 	"sort"
 	"strings"
 
-	"github.com/google/go-cmp/cmp"
+	"github.com/google/go-cmp/cmp" //nolint:depguard
```

Comment:
Would it make sense to turn this into a TODO?

```suggestion
	"github.com/google/go-cmp/cmp" //nolint:depguard // TODO (https://github.com/kubernetes/kubernetes/issues/104821): remove this dependency
```

Same for all other places where it's not okay to use it.

Otherwise what'll happen is that other developers copy-and-paste this entire line without thinking about it...

Reply:
@pohly we have now an active issue https://github.com/kubernetes/kubernetes/issues/104821 and a volunteer in there who wants to help. So i don't want to do a TODO in this many files. Let's try to catch it during reviews. If it starts happening, we can do more to curb this situation. I'd like to watch and see.

Reply:
Fair enough.

Reply:
@pohly could you please `/lgtm` :)